docs/developers/contribute.md

```diff
@@ -3,8 +3,8 @@
 Welcome to the molfeat community! We appreciate everyone's contribution and welcome all. Apart from code contributions, there are various other ways to support the community. If you would like to get involved, please see the [community documentation](../community/get_involved.md) for more details on how to contribute. However you choose to contribute, please be mindful and respect our [code of conduct](https://github.com/datamol-io/molfeat/blob/main/.github/CODE_OF_CONDUCT.md).
 
 !!! note annotate "Get inspired by community contributions"
-  As we love community contributions, we decided to create a dedicate space in our documentation to highlight and celebrate such contributions.
-  Visit [Community Contribitons](../community/contributions.md) to learn more!
+    As we love community contributions, we decided to create a dedicate space in our documentation to highlight and celebrate such contributions.
+    Visit [Community Contribitons](../community/contributions.md) to learn more!
```

Comment:
Those `!!!` are great but too bad markdown editors are not aware of those... Thanks.